storage/libradb/src/ledger_store/mod.rs

```diff
@@ -168,14 +168,14 @@ impl LedgerStore {
         Ok(latest_validator_set.clone())
     }
 
-    fn get_tree_state(
+    pub fn get_tree_state(
         &self,
-        version: Version,
+        num_transactions: LeafCount,
         transaction_info: TransactionInfo,
     ) -> Result<TreeState> {
         Ok(TreeState::new(
-            version,
-            Accumulator::get_frozen_subtree_hashes(self, version + 1)?,
+            num_transactions,
+            Accumulator::get_frozen_subtree_hashes(self, num_transactions)?,
             transaction_info.state_root_hash(),
```

Comment:
Unrelated, but this feels a bit weird -- it passes in `transaction_info` for the purpose of calling `transaction_info.state_root_hash()`...

Reply:
🤣 it makes the call site lines shorter.. 

well, it'll probably make more sense if it takes a &TransactionInfo, which avoids copying a HashValue compared to passing state_root_hash in directly.